packages/cli/src/commands/env/add.ts

```diff
@@ -20,6 +20,7 @@ import { isAPIError } from '../../util/errors-ts';
 type Options = {
   '--debug': boolean;
   '--sensitive': boolean;
+  '--yes': boolean;
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (blocking):** ‏use `--force`

We use `--yes` to choose the default answer for a prompt to the user without actually prompting. The intended behavior of this PR is more in line with how we use the `--force` option.

Can you change this to `--force`?